packages/gamut/src/Form/__tests__/Input-test.tsx

```diff
@@ -0,0 +1,70 @@
+import { StreakIcon } from '@codecademy/gamut-icons';
+import { setupEnzyme } from '@codecademy/gamut-tests';
+
+import { Input } from '../Input';
+
+const renderWrapper = setupEnzyme(Input, {});
+
+describe('Input', () => {
+  it('renders a text input type', () => {
```

Comment:
Nit: these test names make it seem like these situations happen always. Adding a "when" clause would help clarify.

```suggestion
  it('renders a text input type when the type is text', () => {
```